### ISSUE TITLE
new release?

### ISSUE THREAD

**Issue by mika** [sctplab/usrsctp#136]
Release 0.9.3.0 dates back to April 8, 2016 and there have been 190 commits since then, maybe a new release could be published? Thanks! @

**Comment by Akagi201** [comment]
+1, please tag this source code, for easily deploy.

**Comment by linuxmaniac** [comment]
I'm the maintainer of [janus package in Debian](https://tracker.debian.org/pkg/janus) I'm willing to package usrsctp too (is a build-dependency). A new release would be really nice

**Comment by linuxmaniac** [comment]
@tuexen do you have any release plans?

**Comment by randy408** [comment]
A new release would be nice, 0.9.3.0 is over two years old.

**Comment by lgrahl** [comment]
I'd like to bump this one. It's somewhat vital for depending projects to be able to pin to a version instead of a commit, or even worse, to master (like I do currently).

**Comment by philn** [comment]
Yes please publish a new release. I currently (try to) rely on git master but I noticed a forced git push was performed recently... Please try to avoid those, at least.

**Comment by tuexen** [comment]
I'm normally only doing squash and merge. Do you have an example what should not be done?

**Comment by philn** [comment]
`git push --force` should be avoided at all cost. The thumb rule usually is that for public repositories, once you push on a well-known branch (such as master), a force-push should not be allowed.

OTOH, for pull-requests it's totally fine to force-push branches. I often do it after rebasing my local changes on top of the latest changes from master, for instance.

I personally don't like merge commits, I prefer a linear history, thus facilitating bisects. But this is your call :)

**Comment by tuexen** [comment]
> `git push --force` should be avoided at all cost. The thumb rule usually is that for public repositories, once you push on a well-known branch (such as master), a force-push should not be allowed.
I don't think I have done a `git push --force`. So could you point me to a commit, which should have been avoided?
> 
> OTOH, for pull-requests it's totally fine to force-push branches. I often do it after rebasing my local changes on top of the latest changes from master, for instance.
> 
> I personally don't like merge commits, I prefer a linear history, thus facilitating bisects. But this is your call :)

**Comment by philn** [comment]
And another force update...
```
From https://github.com/sctplab/usrsctp
 + 17983b0...aa10d60 master     -> master  (forced update)
Failed to download sources: module libusrsctp: Git commit for branch master is aa10d60bc209028b7dd0370b2d5af9f14051fd23, but expected 17983b032c3f0e33523972e534b18df832922800    
```

**Comment by philn** [comment]
My apologies, I had the flatpak build manifest for this lib mis-configured :)

**Comment by tuexen** [comment]
Will make a release once a couple of issues have been sorted out. Leaving this open until then.

**Comment by jonassmedegaard** [comment]
Quoting Michael Tüxen (2020-01-28 21:26:09)
> Will make a release once a couple of issues have been sorted out. 
> Leaving this open until then.

Great!  Thanks for your work on this!

 - Jonas

-- 
 * Jonas Smedegaard - idealist & Internet-arkitekt
 * Tlf.: +45 40843136  Website: http://dr.jones.dk/

 [x] quote me freely  [ ] ask before reusing  [ ] keep private

**Comment by ncopa** [comment]
> Will make a release once a couple of issues have been sorted out. Leaving this open until then.

What are those issues?

**Comment by tuexen** [comment]
Fixing the issue being addressed by https://github.com/sctplab/usrsctp/pull/417 and three reported issues found by fuzz testing.
